src/test/resources/assets/loot_context_tweaks_test/advancements/test/root.json

```diff
@@ -0,0 +1,23 @@
+{
+	"display": 
+	{
+        "icon": {
+            "item": "minecraft:stone"
+        },
+		"description": "From LootContextTweaksTest",
+		"title": "Test",
+		"background": "minecraft:textures/blocks/stone.png"
+	},
+	"criteria": 
+	{
+           "no": 
+           {
+               "trigger": "minecraft:impossible"
+           }
+     },
+	"rewards": 
+	{
+		"loot": ["minecraft:chests/simple_dungeon"]
+	},
+	"requirements": [["no"]]
```

Comment:
"requirements" can be omitted if all of the criteria are needed, as it is in this case.